project-words.txt

```diff
@@ -8,3 +8,4 @@ Roboto
 subheadline
 linecap
 linejoin
+clsx
```

Comment:
FYI You can add npm script which will add all words with warnings to this list

Reply:
U could use my script: "cspell:apply": "cspell --words-only --unique \"src/**/*.js\" > project-words.txt",

Reply:
But let's use `--gitignore .` instead of `src/**/*.js` to be consistent with 

https://github.com/dev-KPI/messenger-frontend/blob/4e626d2cd7329181f09674e935397c9474a2ce02/package.json#L17

Reply:
I can't find out the final version of the command)

Reply:
```json 
{
  "cspell:apply": "cspell --words-only --unique --gitignore . >> project-words.txt"
}
```

Reply:
> ```json
> {
>   "cspell:apply": "cspell --words-only --unique --gitignore . >> project-words.txt"
> }
> ```

I've used exactly the sme commnd, it doen't include words I've mentioned

Reply:
I'll add this script in another PR